policies/ucs_domain_policies/syslog_policy/terraform/syslog_policy.tf

```diff
@@ -21,22 +39,14 @@ resource "intersight_syslog_policy" "syslog1" {
     min_severity = "emergency"
     object_type  = "syslog.RemoteLoggingClient"
   }
+  
   profiles {
     moid        = var.profile
     object_type = "server.Profile"
   }
-  organization {
-    object_type = "organization.Organization"
-    moid        = var.organization
-  }
 }
 
-variable "organization" {
-   type = string
-   description = "<value for organization>"
- }
-
- variable "profile"{
-   type = string
-   description = "Moid of server.Profile"
- }
\ No newline at end of file
+variable "profile" {
```

Comment:
remove profile here, it is not required we are creating here syslog_policy

Reply:
Removed.